src/components/storefront/CartItem/CartItem.js

```diff
@@ -35,14 +46,18 @@ const CartItem = ({ title, sku, image, price, quantity, lineItemId, checkoutId }
   }, [quantityPickerProps.valueAsNumber]); // eslint-disable-line react-hooks/exhaustive-deps
 
   return (
-    <Flex justifyContent="space-between">
+    <HStack justifyContent="space-between" align="flex-start" spacing={[2, 6]}>
       <Flex>
-        <Flex direction="column">
-          <Link as={PreserveQueryParamsLink} to={`products/${slug}`}>
-            <Image boxSize="200" objectFit="cover" border="1px solid black" src={image} alt={title} />
-          </Link>
-        </Flex>
-        <VStack alignItems="flex-start" w={64} pl={6}>
+        <Link as={PreserveQueryParamsLink} to={`products/${slug}`} flexShrink={0}>
+          <Image
+            boxSize={{ base: '75px', sm: '125px', md: '150px', xl: '200px' }}
+            objectFit="cover"
+            border="1px solid black"
+            src={image}
+            alt={title}
+          />
+        </Link>
+        <VStack alignItems="flex-start" minW="100px" pl={6}>
           <chakra.h4 textStyles="lightCaption" color="brand.gray">
```

Comment:
I just realized this should be `textStyle` instead of `textStyles`, which is causing the SKU to not be all caps, do you mind changing this for me please 😅

Reply:
great catch, fixed in dd4daf13eb